### ISSUE TITLE
modify top-k logic to allow bi-directional access to single collection

### ISSUE THREAD

**Issue by curtislisle** [XDATA-Year-3/EntityAlignLarge#28]
It would be cleaner to have a single top-k collection, instead of two separate ones.  All the current metrics are symmetric, so bidirectional access is OK.  App logic could be changed so a single top-k collection would suffice.

**Comment by jeffbaumes** [comment]
+1 This makes sense to me too. I talked to @ronichoudhury about this - as long as the app logic can be integrated easily this would make our lives easier.

**Comment by curtislisle** [comment]
I agree this is the way to go for this testing application.  I do have the temporary collection-copying scripts now, so we could put it off if we need to. 

Having two collections is definitely a hack for this testcase.  Though we are giving up generality as we optimize to hard-coded, bi-directional references,  so maybe we want to think about how we could keep some flexibility to use with other networks in the future.  ex. ISIL network against itself for QCR.
